Fix link to 2.3-compatible JSON settings

jasonzio opened the pull request:
The link points to the GitHub page for that file; this change replaces it with a link to the raw contents of the file.

azurecla commented:
Hi __@minghan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@minghan 
Good catch! Thank you for your contribution.

--Carolyn
Azure Documentation

jasonzio committed:
Fix link to 2.3-compatible JSON settings

The link points to the GitHub page for that file; this change replaces it with a link to the raw contents of the file.

jasonzio committed:
Improve sample setup

Correct another instance of the JSON file URL
Add Azure CLI command to select the Azure subscription to be used
Suggest installing wget if it's not present
